src/components/sbb-timetable-row/sbb-timetable-row.helper.tsx

```diff
@@ -35,6 +36,23 @@ export const durationToTime = (duration: number): string => {
   return result.join(' ');
 };
 
+export const convertCauseInIconName = (cause: PtSituationCauseEnum): string => {
```

Comment:
I'm not sure whether this should be done in lyne. But all the pearl chain is very near to your backend API, so maybe it's ok for the moment. @kyubisation ?